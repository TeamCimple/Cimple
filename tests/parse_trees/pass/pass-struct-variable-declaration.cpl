struct MyStruct {
	int a;
};

int main() {
	struct MyStruct m;

	return 0;
}
