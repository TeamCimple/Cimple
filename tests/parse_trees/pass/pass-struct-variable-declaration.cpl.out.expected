Struct((int Var(a),), , MyStruct, Constructor(Body:
        CompoundStatement(Declarations:  StatementList: )),, )FuncDecl(Name: Var(main) ReturnType: int Parameters:  Body: CompoundStatement(Declarations: (MyStruct Var(m),),  StatementList: RETURN(Int(0)))) 
